### PR TITLE
Post on coop values

### PR DESCRIPTION
**Pull request by lakikowolfe**
If your pull request is for adding a blog post, please check that you include the following:

- [ ] _Title_ - A short, catchy description to entice readers
- [ ] _Author_ - Your name, which should match your contributor [file](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#adding-yourself-as-a-contributor). If you do not have a contributor file, please feel free to contact one of our team members to assist you
- [ ] _Categories_ - Select 1 at minimum of either Community, Technical, or Science as [describe here](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#what-kind-of-posts-are-appropriate-for-our-blog)
- [ ] _Tags_ - Suggested keywords that can help readers find information from among our blog posts. Please suggest a few tags following [our guidelines](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#tags).
- [ ] _Content_ - This is the body of the blog post! We recommend relatively short posts (no more than five paragraphs), equivalent to a three minute read. Pictures and links to other sites are also great!

The Coop team member reviewing the submission will check for the following:

- [ ] The post is appropriate for the blog; [see guidelines](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#what-kind-of-posts-are-appropriate-for-our-blog)

- [ ] The author is added as a [contributor](https://fredhutch.github.io/coop/contributors/) to the blog

- [ ] The file is in the DRAFT folder

- [ ] A plan for publication (adding `YEAR-MONTH-DAY` to filename and moving to posts folder).

If the pull request is not for adding a blog then delete the content above and briefly describe below the changes proposed in the pull request.

### CODE REVIEW

**Comment by lakikowolfe** [comment]
from convo with Kate: Remove direct links to zoom registration (write what folks can google in centernet)

File: _drafts/dei-coop.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ categories:
...
+Tech and data science spaces tend to be overwhelmingly white. This is true here at Fred Hutch and within The Coop. I am writing this post to express The Coop's firm stance against racial injustice in our community and beyond. We are committed to uplifting and amplifying the voices of Black biologists and data scientists. We are committed to incorporating discussions of ethics into our classes on data science. We are committed to hearing from our community when we make mistakes, or when simply not enough is being done. If you have thoughts, questions, or concerns please feel empowered and encouraged to reach out to either Kate Hertwick or myself (Lauren Wolfe) through Slack, Teams, or email.
```
Comment (k8hertweck) [inline _drafts/dei-coop.md:11]: Hertwick -> Hertweck

File: _drafts/dei-coop.md [hunk 2]
```diff
@@ -6,17 +6,17 @@ categories:
...
 
 ## Get involved at Fred Hutch
 
-Fred Hutch's Office of Diversity, Equity, and Inclusion (ODEI) is holding sessions to engage the Fred Hutch commuity in structured conversation about racisim.
+Fred Hutch's Office of Diversity, Equity, and Inclusion (ODEI) is holding sessions to engage the Fred Hutch commuity in structured conversation about racisim. You can register for either of the following sessions by emailing the ODEI at `diversity@fredhutch.org` and requesting a registration link.
```
Comment (k8hertweck) [inline _drafts/dei-coop.md]: email as `diversity` at `fredhutch.org` (this is how we've included things on the coop website, I think)

File: _drafts/dei-coop.md [hunk 3]
```diff
@@ -6,17 +6,17 @@ categories:
...
 
-* The second session, June 10, is open to all employees. We will facilitate an honest conversation about racism and discuss what we can do to foster a greater sense of inclusion among our community — inside and outside of the Hutch. Register in advance [here](https://zoom.us/meeting/register/tJErce6prjwuHNTYjaSQWFFiTfR78rDe7xEo).
+* The second session, June 10, is open to all employees. We will facilitate an honest conversation about racism and discuss what we can do to foster a greater sense of inclusion among our community — inside and outside of the Hutch. 
 
 Year-round we have an amazing group of organizers running [Diversity Council](https://centernet.fredhutch.org/cn/u/di/erg/diversity-council.html). Diversity Council serves the Fred Hutch community by giving a voice to racial and ethnic minorities, promoting employee engagement through diversity and inclusion educational initiatives, and celebrating successes through inclusive representation in internal and external communications. Join their listserv via the link provided to learn more about upcoming events.
```
Comment (k8hertweck) [inline _drafts/dei-coop.md]: note that this is an internal link, or look to see if there's something on the public FH site

Comment (lakikowolfe) [reply _drafts/dei-coop.md]: good catch thanks!

**Review by lakikowolfe — COMMENTED** [review]
review suggestions have been incorporated

**Pull request merged**